### PR TITLE
Upgrade ECS clusters to use V2 URLs

### PR DESCRIPTION
**Pull request by johnsilva-okta**

### CODE REVIEW

File: link2aws.js [hunk 1]
```diff
@@ -1084,4 +1084,4 @@ if (typeof (require) !== 'undefined' && require.main === module) {
     }
 }
 
-exports.ARN = ARN;
\ No newline at end of file
+exports.ARN = ARN;
```
Comment (fxkr) [inline link2aws.js]: Could you undo the unnecessary whitespace change here?

Other than that, looks good to me. And thanks, I appreciate it!

(Git tip: interactive add (`git add -p`) is quite nice to avoid staging accidental changes.)

Comment (johnsilva-okta) [reply link2aws.js]: Apologies, my VSCode has the setting `Insert Final Newline` enabled. Fixed.

**Comment by fxkr** [comment]
Thank you very much! Merged & released as version 1.0.15.
